test/integration/test_twemcache.py

```diff
@@ -11,12 +13,13 @@ def defineTest(suite, fname, test_type):
     test.load('twemcache/' + fname)
     suite.addTest(test)
 
-def twemcache():
+def twemcache(PmemFlag=False):
```

Comment:
nit:
please use small letter "pmem_flag" instead of PmemFlag (this is how it is done in other files here in function definition)